src/components/Observations/index.tsx

```diff
@@ -1,95 +1,71 @@
-    [setPlantsSitePreferences]
-  );
+  useEffect(() => {
+    if (species === undefined) {
+      dispatch(requestSpecies(selectedOrganization.id));
+    }
+  }, [dispatch, selectedOrganization.id, species]);
 
-  const onPlantingSites = useCallback(
-    (sites: PlantingSite[]) => {
-      if (!sites.length) {
-        history.push(APP_PATHS.HOME);
-      }
-      setPlantingSites(sites);
-    },
-    [setPlantingSites, history]
-  );
+  useEffect(() => {
+    if (plantingSites === undefined) {
+      dispatch(requestPlantingSites(selectedOrganization.id));
+    }
+  }, [dispatch, selectedOrganization.id, plantingSites]);
 
   useEffect(() => {
-    if (plantingSites?.length) {
-      dispatch(
-        requestObservationsResults(
-          selectedOrganization.id,
-          plantingSites.map((site) => ({ id: site.id, timeZone: locationTimeZone.get(site).id }))
-        )
-      );
+    if (species !== undefined && plantingSites !== undefined && observationsResults === undefined) {
+      dispatch(requestObservationsResults(selectedOrganization.id));
     }
-  }, [dispatch, plantingSites, selectedOrganization.id, locationTimeZone]);
+  }, [dispatch, selectedOrganization.id, species, plantingSites, observationsResults]);
 
   useEffect(() => {
-    if (observationsResultsError) {
+    if (observationsResultsError || speciesError || plantingSitesError) {
       snackbar.toastError();
     }
-  }, [observationsResultsError, snackbar]);
+  }, [snackbar, observationsResultsError, speciesError, plantingSitesError]);
+
+  // show spinner while initializing data
+  if (
+    observationsResults === undefined &&
+    observationsResultsError === undefined &&
+    speciesError === undefined &&
+    plantingSitesError === undefined
+  ) {
+    return <CircularProgress sx={{ margin: 'auto' }} />;
+  }
 
-  return (
-    <PlantsPrimaryPage
-      title={strings.OBSERVATIONS}
-      onSelect={onSelect}
-      pagePath={APP_PATHS.PLANTING_SITE_OBSERVATIONS}
-      lastVisitedPreferenceName='plants.observations.lastVisitedPlantingSite'
-      plantsSitePreferences={plantsSitePreferences}
-      setPlantsSitePreferences={onPreferences}
-      allowAllAsSiteSelection={true}
-      onPlantingSites={onPlantingSites}
-      isEmptyState={!plantingSites?.length || !observationsResults?.length}
-    >
-      {observationsResults === undefined && observationsResultsError === undefined ? (
-        <CircularProgress sx={{ margin: 'auto' }} />
-      ) : selectedPlantingSite && observationsResults?.length ? (
-        <ObservationsDataView selectedPlantingSiteId={selectedPlantingSite.id} />
-      ) : (
-        <Card style={{ margin: 'auto' }}>
-          <EmptyStateContent
-            title={strings.OBSERVATIONS_EMPTY_STATE_TITLE}
-            subtitle={[strings.OBSERVATIONS_EMPTY_STATE_MESSAGE_1, strings.OBSERVATIONS_EMPTY_STATE_MESSAGE_2]}
-          />
-        </Card>
-      )}
-    </PlantsPrimaryPage>
-  );
+  return <ObservationsHome />;
```

Comment:
at some point this page will instantiate the other drill down views (based on url params present)